cardano-testnet/src/Testnet/Options.hs

```diff
@@ -30,3 +42,142 @@ defaultTestnetOptions = BabbageTestnetOptions
   , babbageTotalBalance = 10020000000
   , babbageNodeLoggingFormat = NodeLoggingFormatAsJson
   }
+
+
+defaultYamlConfig :: KeyMapAeson.KeyMap Aeson.Value
+defaultYamlConfig =
+  mconcat $ map (uncurry KeyMapAeson.singleton)
+    [
+    -- TODO: Remnants from the Byron era that we can actually eliminate
+    -- and hardcode.
+      ("ApplicationName", "cardano-sl")
+    , ("ApplicationVersion", Aeson.Number 1)
+
+    -- ^ The consensus protocol to use
+    , ("Protocol", "Cardano")
+
+    -- ^ Socket path of the node
```

Comment:
`-- ^` is haddock and it usually is a comment for the line above the comment, not the one below.

We should just `--` instead.